### ISSUE TITLE
Add option to disable adding auto generated audio title metadata

### ISSUE THREAD

**Issue by bdaroz** [FallingSnow/h265ize#40]
When batch converting a bunch of videos, many of them may not be properly tagged for language, or the user may not want additional metadata (which may present in the player) added to the audio stream.

Can you please add an option to disable adding audio title metadata?

Example problem output:
```
[h265ize]: [alert] Audio does not have a title. Title set to "Unknown MP3 (Stereo)".
```

In this example the player would default to playing the audio w/o a label until the metadata is added giving the user a bad UX because of the "Unknown" audio. Yes, one could go through the multitude of videos and ensure they all have proper language and meta tags, but:
```
[h265ize]: [alert] Audio does not have a title. Title set to "English AC3 (6.0 Channel)".
```

For someone who doesn't know what AC3 (or MP3) is, there's no need to add that to a label. They just want to know (if they want to know at all) what the language is.

For my videos the only title metadata that's stored on a per-stream basis is for subtitles to denote SDH subs.

**Comment by FallingSnow** [comment]
> In this example the player would default to playing the audio w/o a label until the metadata is added giving the user a bad UX because of the "Unknown" audio.

You're saying the player would select the stream without a title? But all streams would have a title. Can you elaborate a little more on this.

> For someone who doesn't know what AC3 (or MP3) is, there's no need to add that to a label. They just want to know (if they want to know at all) what the language is.

This is user preference and for those that want to only know what the language is, they can just ignore the `AC3 (6.0 Channel)`.

**Comment by bdaroz** [comment]
No, you misunderstand. The people that view the videos I am re-encoding are... well... not technologically inclined. If they see a label in the player that says "Unknown" for the audio I bet I get a phone call. (Think elderly parent.) At the same time I don't want to have that label include "AC3" "MP3" or anything else (ring, ring). The player will, if there is a language tag, and no title tag, just call it "English", but if there's a title tag it'll use that instead.

Long story short, I have to post-process all the converted videos and strip the title metadata manually.

**Comment by FallingSnow** [comment]
If there is a already title, it is preserved. If there isn't, the title is added as shown above. If you want to remove the `AC3 (6.0 Channel)` section from the generated title you can edit [this line](https://github.com/FallingSnow/h265ize/blob/master/h265ize#L876).

**Comment by bdaroz** [comment]
I've forked the repo - I'll see about doing a pull request, but node.js isn't one of my goto languages.
